abs.yaml

```diff
@@ -0,0 +1,2 @@
+upload_channels:
```

Comment:
```suggestion
channels:
  - sfe1ed40
upload_channels:
```
To get access to opencv-python-headless.

Reply:
`upload_channels` should work also for sourcing the deps, I'll try first without `channels`

Reply:
it works